dockerclient/client.go

```diff
@@ -163,6 +168,7 @@ func (e *ClientExecutor) WithName(name string) *ClientExecutor {
 
 	child := &copied
 	e.Named[name] = child
+	e.Named[fmt.Sprintf("%d", position)] = child
```

Comment:
Why not use strconf.Itoa() it is supposed to be more efficient.

Reply:
Only that I forgot about that function.  Changing it.